test/trento/domain/sap_system/sap_system_test.exs

```diff
@@ -377,7 +378,7 @@ defmodule Trento.SapSystemTest do
           sid: sid,
           db_host: db_host,
           tenant: tenant,
-          instance_number: "00",
+          instance_number: "10",
```

Comment:
It would be nice to put the instance number `00` in the `application_instance_registered_event` explicitly.
I know that it has this value from the factory, but it makes obvious this way that the number is different